src/apps/backend/modules/authentication/types.ts

```diff
@@ -86,3 +86,104 @@ export class UnAuthorizedAccessError extends ApplicationError {
     this.httpStatusCode = HttpStatusCodes.UNAUTHORIZED;
   }
 }
+
+export enum OtpStatus {
```

Comment:
I believe it should be `OTP` instead of `Otp` here and other places. Kindly refactor accordingly.